### PR TITLE
Fix a bug where product images were not scrollable after product refresh

### LINKED ISSUE #12064: After changing product attribute, thumbnails scroll is not working.
Fresh 1.7.5 installation.
After changing an attribute that has other product pictures assigned it it, thumbnails scrolling is not working on quick view or in product page. 

Te reproduce:
1. Go to product page
2. Select an attribute that changes product pictures 
3. Click on cover image to open photos zoom
4. Try to scroll thumbnails by clicking arrows 

issue

PrestaShop version: 1.7.5
PHP version: 7.0

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Product image scroll on click on the image were not working after changing an attribute of the product
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #12064 
| How to test?  | Go on product page which has a lot of images, click on the image, check if arrow scrolls are working at the right, then change an attribute of the product and click again on the image and the arrow scroll should be working

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16863)

### CODE REVIEW

File: themes/classic/_dev/js/product.js [hunk 1]
```diff
@@ -24,6 +24,7 @@
  */
 import $ from 'jquery';
 import prestashop from 'prestashop';
+import ProductSelect from './components/product-select';
```
Comment (Progi1984) [inline themes/classic/_dev/js/product.js:27]: :question: 

```suggestion
import ProductSelect from '@components/product-select';
```

Comment (PierreRambaud) [reply themes/classic/_dev/js/product.js:27]: Not implemented on the classic theme :)

Comment (Progi1984) [reply themes/classic/_dev/js/product.js:27]: Thanks for the answer @PierreRambaud

**Review by Progi1984 — DISMISSED** [review]

**Comment by sarahdib** [comment]
@NeOMakinG this PR need to be rebased on 1.7.7.X branch

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @NeOMakinG ! I still have the issue with your PR, could you check if it's OK for you ?

**Comment by sarahdib** [comment]
@Robin-Fischer-PS you need to clear the cache it's ok for me :)

**Comment by PierreRambaud** [comment]
Thanks @NeOMakinG

**Pull request merged**